config/global.features

```diff
@@ -23,6 +23,7 @@ libpng        = 0
 openssl11     = 0
 python        = 0
 qt            = 0
+qt5           = 0
```

Comment:
qt4 doesn't have this, did I miss it or do we deliberately want qt5 to be different?

Reply:
Maybe, but I'm not sure. My reasoning was that Qt5 isn't a drop in replacement for Qt4 as far as the build system is concerned, but thinking about it now Qt4 wasn't a drop in replacement for Qt3 either.

Reply:
I'm surprised qt4 isn't listed here.  So if you have a build with Qt4 libraries available and don't use -features or default.features, the project that requires qt4 is included?

Reply:
It appears that is the case. Do you want me to add qt4 feature as well?

Also is there a reason why features have to be disabled in this way instead of MPC just assuming a feature is unsupported until explicitly told that it is supported? It seems silly that I can put: `requires += something_fake_or_typo` in a mpc or mpb file and MPC will accept it without question.